docs/hub-deployment-guide/deploy-support/configure-support.md

```diff
@@ -51,19 +51,6 @@ prometheus:
             - <prometheus-domain>
 ```
 
-````{warning}
-If you are deploying the support chart on an AWS cluster, you **must** enable the `cluster-autoscaler` sub-chart, otherwise the node groups will not automatically scale.
-Include the following in your `support.values.yaml` file:
-
-```yaml
-cluster-autoscaler:
-  enabled: true
-  autoDiscovery:
-    clusterName: <cluster-name>
-  awsRegion: <aws-region>
-```
-````
```

Comment:
@consideRatio, apparently it was still documented, I might have missed it when deploying dandi. Anyway, I've removed it now that we autogenerate it via the deployer.